vars/patchDockerImage.groovy

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+void call(Map args = [:]) {
+    def lib = library(identifier: 'jenkins@main', retriever: legacySCM(scm))
```

Comment:
Are we still using lib definition in the lib file? @gaiksaya I thought we have moved away from that for a long time?

Reply:
We did not move away from this. Its still being used in most of the lib. The whole version increment workflow is for replacing this `main` to appropriate version when we bump. Also this is required to used lib within lib.